README.md

```diff
@@ -10,6 +10,29 @@ Expected environment:
   - Node Feature Discovery (NFD) Operator
   - Kernel Module Management (KMM) Operator ([Redhat docs](https://docs.openshift.com/container-platform/4.12/hardware_enablement/kmm-kernel-module-management.html), [codebase docs](https://kmm.sigs.k8s.io/documentation/deploy_kmod/))
 
+#### Base images required
+
+For onload & sfc:
+* `DTK_AUTO`
```

Comment:
nit: Maybe list it as the last item (followed by the explanation of how to compute it)?